### ISSUE TITLE
MkDocs theme - Navigation - menus - multple rows - overlaps content

### ISSUE THREAD

**Issue by randomessence** [mkdocs/mkdocs#210]
This is an issue with the actual bootstrap / bootswatch themes and not mkdocs itself.

I will reference another similar issue and what that dev did rather than explain it twice. The problem is exactly the same so the below link/issue applies to mkdocs and the applied themes.

https://github.com/Dynalon/mdwiki/issues/36

His fix ( it may not be applicable but maybe something there is helpful)

https://github.com/Dynalon/mdwiki/commit/242d5914dddf08fb1189cdad77583489ad35b33b

tl:dr 

If you have enough section entries to cause a second or more rows in the navigation bar it will overlap the main content rather than push it down until the screen is resized to the mobile version and the Nav bars changes to the mobile version.

**Comment by d0ugal** [comment]
To add a visual aid, as I understand, this is the problem. I can't scroll up, the content that is hidden by the nav is lost.

![](https://dl.dropboxusercontent.com/u/14255217/Screenshots/Screenshot%202015-04-09-21.11.30.Google-chrome.png)

**Comment by d0ugal** [comment]
For now, I'm inclined to say this is a limitation of the theme. If you really need that many pages and they can't be nested, either the readthedocs theme or a custom theme would be needed.

**Comment by d0ugal** [comment]
Okay, I'm going to put this down to a limitation of the theme. Now we have landed a fix for #6, users should nest the pages more or use a different them - width like this will never work well in these type of themes.

**Comment by d0ugal** [comment]
I can't remember why I reopened this, I agree with my [closing comment](https://github.com/mkdocs/mkdocs/issues/210#issuecomment-100562959) still (or again?) so closing for now.

**Comment by tomchristie** [comment]
Yup I'd agree with that assessment.

**Comment by ProfYaffle** [comment]
What triggers the shift to the mobile menu - is it just the viewport size? Any way to force that as an override? IIRC, once it switches to mobile then the problem vanishes as the menu compacts down, so that may be a valid workaround...
